doc/changes/changes_0.2.0.md

```diff
@@ -1,10 +1,18 @@
-# bucketfs-document-files-virtual-schema 0.2.0, released 2020-10-XX
+# project-keeper-maven-plugin 0.2.0, released 2020-10-13
      
-Code name: 
+Code name: POM file validation
     
 ## Features / Enhancements
 
 * #8: Pom-file validation
+* #13: Added templates for:
+ * ossindex-maven-plugin
```

Comment:
I did not list all contents of the default template since it's quite a lot.

Reply:
Well, it's not a perfect solution if I come to you every time to ask if the project keeper check this or that or not. 
Or even worse I'll need to search for this information in the code. Please, invest some time and list them. It will make our lives easier.